msprime/mutations.py

```diff
@@ -322,8 +325,12 @@ class MicrosatMutationModel(MatrixMutationModel):
     and are unrelated to repeat length
     (e.g., the repeat itself can be dinucleotide, tri-nucleotide, etcetera).
 
-    The default values for ``lo`` and ``hi`` are chosen based on
-    FIXME.
+    .. warning::
+        The default values for ``lo`` and ``hi`` are chosen arbitrarily.
+        The default of ``lo=2`` is chosed because repeat number need be greater
+        than for it to be considered a repeat. Please ensure
```

Comment:
This justification doesn't make sense to me - if it's polymorphic in the population, and *some* individuals have > 1 copy, then it's a repeat. For instance, it could make sense to parameterize things so that `lo=1` and `ancestral_state` is always `1`, and so it's only variable sites that start getting repeat numbers. Because: everything is a *potential* repeat. (Not saying we should do it that way.)

Reply:
sure, absolutely. I was thinking more about these particular satellite repeats that are ascertained as STRs or whatever, but you are definitely correct.

Reply:
and it would be easy enough to change the default to `lo=1`

Reply:
btw you can also look through human microsats here-- because of the ascertainment, all of these have min allele lengths at 3, 4, 5 https://strbase.nist.gov/fbicore.htm

Reply:
Right, but we're simulating the mutation process, not the ascertainment? I vote to change to `lo=1`.